[Blog] DockerCon 2022 Talk with HashiCorp | CU-2g4fptm (#2765)
layer5io/layer5: src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx

---
title: "Extending the Docker Compose Experience to Service Mesh"
date: 2022-05-10 10:30:05 -0530
author: Gaurav Chadha
thumbnail: ./DC22-talk-HashiCorp.png
description: "DockerCon 2022 talk with HashiCorp and MeshMap Demo"
type: Blog
category: "Docker"
tags:
 - Docker
 - Meshery
published: true
---

import { BlogWrapper } from "../Blog.style.js";
import Blockquote from "../../../reusecore/Blockquote";
import slidesTAG from "../../events/kubecon-china-2021/cncf-tag-network-and-service-mesh-working-group-kubecon-china-2021-lee-calcote-ken-owens.pdf";
import { Link } from "gatsby";
import DDE from './docker-extension-meshery.png';
import developer from './developers-need.png';
import dashboard from './dashboard.png';
import designer from './designer-1.png';
import visualizer from './viz-1.png';
import thankyou from './thankyou.png';

<BlogWrapper>

<div class="intro ">
<p>

>     <Link to="/community/members/lee-calcote">Lee Calcote</Link> and <Link to="/community/members/nic-jackson"> Nic Jackson </Link> gave a presentation entitled Intro and Deep Dive into Docker Extension for Meshery at DockerCon 2022.

Reviewer: ```suggestion
    <Link to="/community/members/lee-calcote">Lee Calcote</Link> and <Link to="/community/members/nic-jackson">Nic Jackson</Link> gave a presentation entitled <i>Extending the Docker Compose Experience to Service Mesh</i> at DockerCon 2022.
```

</p>
</div>

<p>

>     The Docker Extension for Meshery extends Docker Desktop's position as the cloud native developer's go-to Kubernetes environment with easy access to the next

Reviewer: ```suggestion
    The <Link to="/docker-extension-meshery">Meshery Docker Extension</Link> extends Docker Desktop's position as the cloud native developer's go-to Kubernetes environment with easy access to the next
```

    <strong> Designer Mode</strong> It can design a service mesh deployment with application and Envoy filter from scratch. Customize a service mesh deployment with application and Envoy filter from pattern.
</p>
       <p>
       <img src={visualizer} alt="MeshMap Designer" style="width:50%;float:left;padding-right:30px;" />
     The <strong>Visualizer Mode</strong> examine a visual topology of Kubernetes cluster and its services. View and search log streams from your pod's containers. Connect an interactive terminal to instances of your containers.
       </p>

<p>
    To Lee's demo, Nic also added in the context of developer experience that MeshMap also makes it easy to do the job and with Meshery extension it makes it super easy to write and configure code without being worried for the developer environment.
    </p>
    <p>
       These Docker Extensions are so powerful that it allows you to do multiple tasks without leaving Docker Desktop. 
        </p>

<p><strong>Lee Calcote and Nic Jackson packed a great deal of information in this talk. The Meshery Extension is now out! Try now, and Share your Experience </strong>

> <a href="https://layer5.io/service-mesh-management/meshmap">Apply for MeshMap Beta Program</a>

Reviewer: ```suggestion
<Link to="/service-mesh-management/meshmap">Apply for MeshMap Beta Program</Link>
```
